util/text.jsx

```diff
@@ -2,6 +2,23 @@ import React from 'react';
 
 const escapeRegex = (text) => `(${text?.replace(/[/\-\\^$*+?.()|[\]{}]/g, '\\$&')})`;
 
+const getIndices = ({ i, regexText, group, indices, index }) => {
+  let start;
+  let end;
+
+  if (indices === null) {
+    start = regexText.indexOf(group) + index;
+    end = start + group.length;
+  } else {
+    // eslint-disable-next-line prefer-destructuring
+    start = indices[i + 1][0];
+    // eslint-disable-next-line prefer-destructuring
+    end = indices[i + 1][1];
+  }
+
+  return { start, end };
+};
+
```

Comment:
A suggestion, if you took my approach above
```suggestion
const getIndices = ({ i, regexText, group, index }) => { 
  const start = regexText.indexOf(group) + index;
  const end = start + group.length;
  return [start, end];
}
```